packages/aws-rfdk/lib/deadline/lib/sep-configuration.ts

```diff
@@ -0,0 +1,403 @@
+   */
+  readonly spotFleetOptions: SEPConfigurationProperties;
+}
+
+/**
+ * This construct configures Spot Event Plugin by connecting to the render queue and executing requests against it.
+ * To provide this functionality, this construct will create an AWS Lambda function that is granted the ability
+ * to connect to the render queue. This lambda is run automatically when you deploy or update the stack containing this construct.
+ * Logs for all AWS Lambdas are automatically recorded in Amazon CloudWatch.
+ *
+ * Resources Deployed
+ * ------------------------
+ * - An AWS Lambda that is used to connect to the render queue, and save Spot Event Plugin configurations.
+ * - A CloudFormation Custom Resource that triggers execution of the Lambda on stack deployment, update, and deletion.
+ * - An Amazon CloudWatch log group that records history of the AWS Lambda's execution.
+ *
+ * Security Considerations
+ * ------------------------
+ * - The AWS Lambda that is deployed through this construct will be created from a deployment package
+ *   that is uploaded to your CDK bootstrap bucket during deployment. You must limit write access to
+ *   your CDK bootstrap bucket to prevent an attacker from modifying the actions performed by this Lambda.
+ *   We strongly recommend that you either enable Amazon S3 server access logging on your CDK bootstrap bucket,
+ *   or enable AWS CloudTrail on your account to assist in post-incident analysis of compromised production
+ *   environments.
+ * - The AWS Lambda function that is created by this resource has access to both the certificates used to connect to the render queue,
+ *   and the render queue port. An attacker that can find a way to modify and execute this lambda could use it to
+ *   execute any requets against the render queue. You should not grant any additional actors/principals the ability to modify
+ *   or execute this Lambda.
+ */
+export class SEPConfigurationSetup extends Construct {
+  constructor(scope: Construct, id: string, props: SEPConfigurationSetupProps) {
+    super(scope, id);
+
+    // We do not check the patch version, so it's set to 0.
+    const version = new Version([props.version.majorVersion, props.version.minorVersion, props.version.releaseVersion, 0]);
+    const minimumVersion: Version = new Version([10, 1, 12, 0]);
+
+    if (version.isLessThan(minimumVersion)) {
+      throw new Error(`Minimum supported Deadline version for ${this.constructor.name} is ` +
+      `${minimumVersion.majorVersion}.${minimumVersion.minorVersion}.${minimumVersion.releaseVersion}. ` +
+      `Received: ${version.majorVersion}.${version.minorVersion}.${version.releaseVersion}.`);
+    }
+
+    const region = Stack.of(this).region;
+    const openSslLayerName = 'openssl-al2';
+    const openSslLayerArns: any = ARNS[openSslLayerName];
+    const openSslLayerArn = openSslLayerArns[region];
+    const openSslLayer = LayerVersion.fromLayerVersionArn(this, 'OpenSslLayer', openSslLayerArn);
```

Comment:
Again copy/paste from similar constructs in RFDK :)
Simplified it to this:
```
const openSslLayerArns: any = ARNS['openssl-al2'];
const openSslLayer = LayerVersion.fromLayerVersionArn(this, 'OpenSslLayer', openSslLayerArns[region]);
```

We still need to explicitly set `ARNS['openssl-al2']` to `any`, otherwise Typescript deduces the type and complains that there is no indexing operator when calling `[region]`.